src/libs/actions/Report.js

```diff
@@ -323,6 +333,81 @@ function fetchChatReportsByIDs(chatList) {
         });
 }
 
+/**
+ * Given IOU and chat report ID fetches most recent IOU data from API and update stored data in Onyx.
+ *
+ * @param {Number} iouReportID
+ * @param {Number} chatReportID
+ */
+function updateIOUReportData(iouReportID, chatReportID) {
+    API.Get({
+        returnValueList: 'reportStuff',
+        reportIDList: iouReportID,
+        shouldLoadOptionalKeys: true,
+        includePinnedReports: true,
+    }).then((response) => {
+        if (!response) {
+            return;
+        }
+
+        if (response.jsonCode !== 200) {
+            console.error(response.message);
+            return;
+        }
+
+        const iouReportData = response.reports[iouReportID];
+        if (!iouReportData) {
+            console.error(`No iouReportData found for reportID ${iouReportID}`);
+            return;
+        }
+        const iouReportObject = getSimplifiedIOUReport(iouReportData, chatReportID);
+        const chatReportObject = {
+            hasOutstandingIOU: iouReportObject.stateNum === 1
+                && iouReportObject.total !== 0,
+            iouReportID,
+        };
+        if (!chatReportObject.hasOutstandingIOU) {
+            chatReportObject.iouReportID = null;
+        }
+        const iouReportKey = `${ONYXKEYS.COLLECTION.REPORT_IOUS}${iouReportObject.reportID}`;
+        const reportKey = `${ONYXKEYS.COLLECTION.REPORT}${chatReportID}`;
+        Onyx.merge(iouReportKey, iouReportObject);
+        Onyx.merge(reportKey, chatReportObject);
+    });
+}
+
+/**
+ * Given chat report ID finds active IOU report ID via GetIOUReport API call
+ *
+ * @param {Number} chatReportID
+ * @returns {Promise}
+ */
+function fetchIOUReportID(chatReportID) {
+    const chatReport = lodashGet(allReports, chatReportID);
+    const participants = chatReport.participants;
+    if (participants.length !== 1) {
+        console.error('chatReport doesn\'t have any participant.');
+        return;
+    }
```

Comment:
Following up from my previous comment, group reports will hit this line but should not be seen as an error. 

Lets add another condition to catch reports with multiple participants and simply return.